index.html

```diff
@@ -54,5 +54,5 @@ <h3 class="text-muted">Project name</h3>
       </div>
 
     </div> <!-- /container -->
-<script src="http://code.jquery.com/jquery-2.1.1.min.js"></script>
+<!-- <script src="https://code.jquery.com/jquery-2.1.1.min.js"></script> -->
```

Comment:
I wasn't so sure if bootstrap would even work  without jquery, but it seems that it'll luckily work if that bootstrap.js is gone. So poof it all away for me :champagne:

> On 26 Jan 2016, at 16:56, Michael Kühnel <notifications@github.com> wrote:
> 
> In index.html:
> 
> > @@ -54,5 +54,5 @@ <h3 class="text-muted">Project name</h3>
> >        </div>
> >  
> >      </div> <!-- /container -->
> > -<script src="http://code.jquery.com/jquery-2.1.1.min.js"></script>
> > +<!-- <script src="https://code.jquery.com/jquery-2.1.1.min.js"></script> -->
> It’s still there.Just with comments. Get that shit outta here!!!!!111one
> 
> —
> Reply to this email directly or view it on GitHub.
>